drivers/platform/linux/linux_gpio.h

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ *   @file   linux/linux_gpio.h
```

Comment:
For consistency it should be gpio_extra.h

Reply:
We shouldn't do it anymore since now, multiple types of GPIO controllers can be instantiated in the same project.